packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1221,6 +1317,26 @@ export default class TradeStore extends BaseStore {
             }
             if (this.is_accumulator) this.resetAccumulatorData();
 
+            // Sometimes the initial barrier doesn't match with current barrier choices received from API.
+            // When this happens we want to populate the list of barrier choices to choose from since the value cannot be specified manually
```

Comment:
@henry-deriv i think this block of code ended up here by accident during conflict fixing, didn't it? we don't need this block. But you will need to add `?? {}` below at line 1356: `const { barrier_choices, max_stake, min_stake } = response.error.details ?? {};`

Reply:
removed